content/enterprise_influxdb/v1.9/administration/configure/config-data-nodes.md

```diff
@@ -356,7 +356,7 @@ Environment variable: `INFLUXDB_DATA_MAX_VALUES_PER_TAG`
 
 ### TSI (`tsi1`) index settings
 
-#### `max-index-log-file-size = "1m"`
+#### `max-index-log-file-size = "131072"  # 1m`
```

Comment:
The code that parses this is here:

```
	// Parse unit of measure
	suffix := text[len(sizeText)-1]
	if !unicode.IsDigit(rune(suffix)) {
		switch suffix {
		case 'k', 'K':
			mult = 1 << 10 // KiB
		case 'm', 'M':
			mult = 1 << 20 // MiB
		case 'g', 'G':
			mult = 1 << 30 // GiB
		default:
			return fmt.Errorf("unknown size suffix: %c (expected k, m, or g)", suffix)
		}
		sizeText = sizeText[:len(sizeText)-1]
	}
```

These are standard SI prefixes, so this should be 

```
#### `max-index-log-file-size = "1000000" # 1m`
```